### PR TITLE
Rw merge

### PR DESCRIPTION
**Pull request by ReidWilliams**
Style pass on `Navigation.vue` and `FileUploader.vue`.

I changed `CreateNewListing.vue` but am still in the process of completing that, but wanted to merge progress so far.

I renamed `listing.sass` to `create-new-listing.sass` to match name of component. I want to be strict about components only importing one non-global sass file, having non-global sass files only be used by one component, and have file path and name match between component and sass file.

### CODE REVIEW

File: src/views/CreateNewListing.vue [hunk 1]
```diff
@@ -4,18 +4,10 @@
...
-          </div>
-          <div class="tile">
-            <FileMetadata/>
-          </div>
-        </div>
-      </div>
+    <div v-if="isReady" class="container new-listing-container">
+      <FileUploader />
+      <FileLister />
+      <FileMetadata/>
```
Comment (MelvinMallari) [inline src/views/CreateNewListing.vue]: Nice cleanup

File: src/assets/style/components/file-uploader.sass [hunk 2]
```diff
@@ -2,61 +2,20 @@
 
 .file-uploader
```
Comment (MelvinMallari) [inline src/assets/style/components/file-uploader.sass]: Would be a good idea to throw a `cursor: pointer` property here as well to communicate clickability

File: src/views/CreateNewListing.vue [hunk 3]
```diff
@@ -44,8 +36,7 @@ import FileMetadata from '@/components/listing/FileMetadata.vue'
 
 import Dropzone from 'dropzone'
 
-import '@/assets/style/views/list.sass'
-import '@/assets/style/components/file-uploader.sass'
+import '@/assets/style/views/create-new-listing.sass'
```
Comment (MelvinMallari) [inline src/views/CreateNewListing.vue]: I agree with the convention. Possibly a good amount of these changes moving forward

**Comment by ReidWilliams** [comment]
I forgot to fix tests, I'll fix and resubmit.

**Comment by ReidWilliams** [comment]
@MelvinMallari updated tests.

**Pull request merged**